src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -295,18 +306,24 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
 
   /**
    * True if date is >= the minimum (ie, today) and <= the maximum (ie, the 10th day).
-   * This is used for Vue form validation (in Date Rules above).
+   * This is used for Vue form validation (in date rules above).
    */
   private isValidDateRange (minDate: Date, maxDate: Date, dateStrToValidate: string): boolean {
-    if (!dateStrToValidate) { return true }
-    // Input is in the format of MM dd, yyyy - only compare year/month/day (ignore time)
-    const utcDateStr = new Date(dateStrToValidate + ' 00:00 UTC').toISOString().split('T')[0]
-    const pstDate = this.yyyyMmDdToDate(utcDateStr)
-    return (pstDate >= minDate && pstDate <= maxDate)
+    if (!dateStrToValidate) return true
+    const date = this.mmmDdYyyyToDate(dateStrToValidate)
+    return (date >= minDate && date <= maxDate)
+  }
+
+  get resolutionTextRules (): Array<Function> {
+    return [
+      v => (v && v.trim().length > 0) || 'Resolution text is required',
+      v => (v && v.length <= this.MAX_RESOLUTION_TEXT_LENGTH) || 'Maximum characters exceeded',
+      v => (v && /^([\w\s$&+,:;=?@#|'<>.^*()%!-\\"]*)$/g.test(v)) || 'Invalid characters'
+    ]
   }
 
   /** Called to update resolution date. */
-  async onResolutionDateSync (val: string): Promise<void> {
+  protected async onResolutionDateSync (val: string): Promise<void> {
```

Comment:
This is not called externally so it shouldn't be public (default), and it shouldn't be private because it's called from the template.